docs/tutorials/data/cta.ipynb

```diff
@@ -37,7 +37,14 @@
   {
    "cell_type": "code",
    "execution_count": null,
-   "metadata": {},
+   "metadata": {
```

Comment:
@bultako Is there anything we can do about this? There is auto-generated meta data added, probably dependent non the jupyter or python version. This is a bit annoying...

Reply:
I decided to remove the `execution` meta data keyword during stripping https://github.com/gammapy/gammapy/commit/370fc3e67b8ed2a573cf993f4310710cb737b5cc @AtreyeeS Please rebase and update the PR. Thanks!